src/view3d.cpp

```diff
@@ -311,37 +461,42 @@ void View3D::paintGL()
 //        s[barSize] = '|';
 //    tfm::printfln("%12f %4d %s", quality, frameTime, s);
 
-    m_incrementalFramebuffer->release();
-    QGLFramebufferObject::blitFramebuffer(0, QRect(0,0,width(),height()),
-                                          m_incrementalFramebuffer.get(),
-                                          QRect(0,0,width(),height()),
-                                          GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
+    // TODO: this should really render a texture onto a quad and not use glBlitFramebuffer
```

Comment:
Right, so that lets you render the point cloud depth incrementally to a depth texture.  However, what we want to do here is copy that depth texture into the driver controlled backbuffer, followed by rendering extra stuff like bounding boxes to the normal backbuffer, with depth testing turned on.  I still don't see how to do that in a natural way.